### PR TITLE
biz_EN.md translated

### PR DESCRIPTION
**Pull request by inov8ru**

### CODE REVIEW

File: biz_EN.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+However, a truly unique and interesting business in VIZ using social capital can be built around creating applications that provide *context for awards*, and receiving *beneficial payments* from users of these applications. Let's try to decipher it.
+
+## Context of the award
+
+In its simplest form , the interaction of people in VIZ is described by the phrase:
+
+> John rewarded Mike (that is, increased his social capital).
+
+![первый пример](https://raw.githubusercontent.com/vizplus/viz-biz/main/1.jpg)
```
Comment (chiliec) [inline biz_EN.md]: Why alternative text in russian?

File: biz_EN.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+# How can you create an unique business in VIZ?
```
Comment (chiliec) [inline biz_EN.md]: "a unique", not "an"

**Review by chiliec — CHANGES_REQUESTED** [review]

File: biz_EN.md (hunk 1 above)
Comment (inov8ru) [reply biz_EN.md]: > Why alternative text in russian?

Fixed.

File: biz_EN.md (hunk 2 above)
Comment (inov8ru) [reply biz_EN.md]: Fixed.

**Pull request merged**